index.html

```diff
@@ -24,14 +24,20 @@
     <li><p>Other light duties, as assigned</p></li>
   </p>
   </ul>
+<div class="greenhead">What time of day and how many times a week will each facility need to be serviced?</div>
```

Comment:
I would break these out into the [yaml frontmatter]() and format them all in one go with a {% for question in questions %} for example, this will make it much easier to add them in the future.